src/modules/shortcut_guide/overlay_window.cpp

```diff
@@ -503,15 +505,33 @@ void D2DOverlayWindow::hide_thumbnail() {
 
 void D2DOverlayWindow::render(ID2D1DeviceContext5* d2d_dc) {
   if (!winkey_held() || is_start_visible()) {
-    hide();
-    instance->was_hidden();
-    return;
+	  auto current_anim_value = animation.value(Animation::AnimFunctions::LINEAR);
+	  if (!hiding) { // when user is done viewing the overlay
+		  animation.reset(anim_time * current_anim_value);
```

Comment:
Please use spaces for indentation.

Reply:
What is the point of multiplying the `anim_time` by `current_anim_value`? If you want to "reverse" the animation before it is completed, you should also change the `start` and `stop` values (there is a second `reset` overload for that)